doc/UsersGuide/BuildingRunningTesting/tutorials/tutorial1.inc

```diff
@@ -0,0 +1,454 @@
+.. _fcst1:
+
+Sample Forecast #1: Severe Weather Over Indianapolis
+=======================================================
+
+**Objective:** Modify physics options and compare forecast outputs for similar experiments using the graphics plotting task. 
+
+Weather Summary
+--------------------
+
+A surface boundary associated with a vorticity maximum over the northern Great Plains moved into an unstable environment over Indianapolis, which led to the development of isolated severe thunderstorms before it congealed into a convective line. The moist air remained over the southern half of the area on the following day. The combination of moist air with daily surface heating resulted in isolated thunderstorms that produced small hail. 
+
+**Weather Phenomena:** Numerous tornado and wind reports (6/15) and hail reports (6/16)
+
+   * `Storm Prediction Center (SPC) Storm Report for 20190615 <https://www.spc.noaa.gov/climo/reports/190615_rpts.html>`_
+   * `Storm Prediction Center (SPC) Storm Report for 20190616 <https://www.spc.noaa.gov/climo/reports/190616_rpts.html>`_
+
+.. figure:: https://github.com/ufs-community/ufs-srweather-app/wiki/Tutorial/IndySevereWeather18z.gif
+   :alt: Radar animation of severe weather over Indianapolis on June 15, 2019 starting at 18z. The animation shows areas of heavy rain and tornado reports moving from west to east over Indianapolis. 
+
+   *Severe Weather Over Indianapolis Starting at 18z*
+
+Data
+-------
+
+On :srw-wiki:`Level 1 <Supported-Platforms-and-Compilers>` systems, users can find data for the Indianapolis Severe Weather Forecast in the usual input model data locations (see :numref:`Section %s <DataLocations>` for a list). The data can also be downloaded from the `UFS SRW Application Data Bucket <https://noaa-ufs-srw-pds.s3.amazonaws.com/index.html>`_. 
+
+   * FV3GFS data for the first forecast (``control``) is located at: 
+   
+      * https://noaa-ufs-srw-pds.s3.amazonaws.com/index.html#develop-20240618/input_model_data/FV3GFS/grib2/2019061518/
+
+   * HRRR and RAP data for the second forecast (``test_expt``) is located at: 
+      
+      * https://noaa-ufs-srw-pds.s3.amazonaws.com/index.html#develop-20240618/input_model_data/HRRR/2019061518/
+      * https://noaa-ufs-srw-pds.s3.amazonaws.com/index.html#develop-20240618/input_model_data/RAP/2019061518/
```

Comment:
For the release, should we point to `develop`, or `experiment-user-cases/release-public-v3.0.0` for the release documentation?